Mathlib/Algebra/Algebra/Subalgebra/Basic.lean

```diff
@@ -711,6 +729,22 @@ theorem sInf_toSubsemiring (S : Set (Subalgebra R A)) :
     (sInf S).toSubsemiring = sInf (Subalgebra.toSubsemiring '' S) :=
   SetLike.coe_injective <| by simp
 
+open Subalgebra in
+@[simp]
+theorem sSup_toSubsemiring (S : Set (Subalgebra R A)) (hS : S.Nonempty) :
+    (sSup S).toSubsemiring = sSup (toSubsemiring '' S) := by
```

Comment:
Similarly `iSup` on the right here looks nicer

Reply:
I'm not sure about this. The existing result `sInf_toSubsemiring` uses `sInf` on RHS. For consistency, I think this should also use `sSup` instead of `iSup`.